### ISSUE TITLE
Doesn't work with editorjs 2.29.0

### ISSUE THREAD

**Issue by orlando-labs** [kommitters/editorjs-tooltip#122]
Hello! Thanks for your work!

Editorjs v2.29.0 catches the exception
```
TypeError: Argument 1 ('target') to MutationObserver.observe must be an instance of Node
```
and rejects the tool installation/usage.

<img width="1469" alt="Снимок экрана 2024-03-18 в 16 39 31" src="https://github.com/kommitters/editorjs-tooltip/assets/73169645/bf974154-15fd-4116-8cce-2194e1dff183">

**Comment by MarioRodriguezS** [comment]
@orlando-labs Thanks a lot for reporting the issue. It has been solved and is available in v1.2.1. Please let us know if the solution was useful to you.

**Comment by orlando-labs** [comment]
Thanks a lot @MarioRodriguezS. I confirm v1.2.1 works
